### PR TITLE
Log successful synchronization.

### PR DESCRIPTION
**Pull request by bbannier**

### CODE REVIEW

**Review by rsmmr — COMMENTED** [review]
This is good, one more idea though: I think all the the sync logs would git better into `spicy-verbose` because (1) that gives the context of where parsing it at, and (2) "spicy" is mean primarily for understanding parsing results (vs. parsing mechanics). How about changing that here as well?

**Comment by bbannier** [comment]
> This is good, one more idea though: I think all the the sync logs would git better into `spicy-verbose` because (1) that gives the context of where parsing it at, and (2) "spicy" is mean primarily for understanding parsing results (vs. parsing mechanics). How about changing that here as well?

Done.

Just to make sure, do we consider such a change breaking so that this patch should not be backported?

File: .update-changes.cfg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-new_commit_msg="Update CHANGES. [skip ci]"
+new_commit_msg="Update CHANGES."
```
Comment (rsmmr) [inline .update-changes.cfg:1]: Unintended change?

**Review by rsmmr — DISMISSED** [review]

**Comment by rsmmr** [comment]
> Just to make sure, do we consider such a change breaking so that this patch should not be backported?

I'd generally exclude debugging output from any stability commitments, but still no need to backport I think; not a bugfix nor crucial to have in other ways.

File: .update-changes.cfg (hunk 1 above)
Comment (bbannier) [reply .update-changes.cfg:1]: Yes, this slipped in when I cut the release.

**Pull request merged**